src/fty_email_server.cc

```diff
@@ -363,10 +366,10 @@ fty_email_server (zsock_t *pipe, void* args)
                 bool sent_ok = false;
                 try {
                     if (zmsg_size (zmessage) == 1) {
-                        char *body = zmsg_popstr (zmessage);
-                        log_debug ("%s:\tsmtp.sendmail (%s)", name, body);
+                        std::string body = getIpAddr();
+                        body += zmsg_popstr (zmessage);
```

Comment:
Memory leak, the result of `zmsg_popstr` must be freed with `zstr_free` or, preferably, be put inside a `ZstrGuard`.